### PR TITLE
Remove deprecated `statuses_before_force_delete` field

### LINKED ISSUE #2363: Reapplying statuses on ForceDelete removal can end up with invalid expired status
there are two similarly named columns in the domains table
statuses_before_force_delete
admin_store_statuses_history element in the json_statuses_history jsonb column

when is admin_store_statuses_history saved and updated?
consider combining the two elements into one

There was an issue with 4 domains so far that had expired status saved in admin_store_statuses_history and ok status saved in the statuses_before_force_delete column. When FD was removed system restored expired status for the domain instead of ok even though the valid_to date was 6 months away.

### PR DESCRIPTION
**Pull request by thiagoyoussef**
close #2363

### CODE REVIEW

File: db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemoveStatusesBeforeForceDeleteFromDomains < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :domains, :statuses_before_force_delete
```
Comment (OlegPhenomenon) [inline db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb:3]: Please google one method. I don't remember the name, but I remember exactly what the method was. The idea is that this method checks whether the given column exists or not. If it doesn't exist, it skips the deletion and if it does, it deletes

**Review by OlegPhenomenon — CHANGES_REQUESTED** [review]

File: db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb (hunk 1 above)
Comment (thiagoyoussef) [reply db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb]: `column_exists?`

Comment (OlegPhenomenon) [reply db/migrate/20220524130709_remove_statuses_before_force_delete_from_domains.rb]: yep

**Comment by thiagoyoussef** [comment]
Staging test is failing because github packages is not working https://www.githubstatus.com/

**Comment by OlegPhenomenon** [comment]
> Staging test is failing because github packages is not working https://www.githubstatus.com/

I was already thinking of asking Martin what the problem was with the deploying on the staging :D

**Pull request merged**